### PR TITLE
MCOL-5618: Add backup, restore, backup-dbrm, restore-dbrm commands for mcs cli tool.

### PR DESCRIPTION
**Pull request by mariadb-AlanMologorsky**
- [x] [add] mcs_cluster_tool/helpers.py file with cook_sh_arg function inside 
- [x] [add] MCS_BACKUP_MANAGER_SH to mcs_cluster_tool/constants.py
- [x] [add] backup and restore commands to "mcs" Typer APP

### CODE REVIEW

File: cmapi/CMakeLists.txt [hunk 1]
```diff
@@ -84,6 +84,9 @@ INSTALL(FILES mcs_aws
 INSTALL(FILES mcs_gsutil
             PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ
             DESTINATION ${BIN_DIR})
+INSTALL(FILES scripts/mcs_backup_manager.sh
```
Comment (drrtuy) [inline cmapi/CMakeLists.txt:87]: mcs_backup_manager.sh doesn't have any hardcoded paths,e.g. datadir path that must be templated @mariadb-AllenHerrera  ?

**Review by drrtuy — APPROVED** [review]
untested ACK

**Comment by drrtuy** [comment]
@mariadb-AlanMologorsky You need to clone this into stable-23.10 b/c the release ver is 23.10.2 that is coming.

File: cmapi/CMakeLists.txt (hunk 1 above)
Comment (mariadb-AllenHerrera) [reply cmapi/CMakeLists.txt:87]: there are some hardcoded defaults, many are parameterized but some like DBRM_PATH are fixed depending if S3 or localstorage ATM
   # Fixed Paths
   MARIADB_PATH="/var/lib/mysql"
   CS_CONFIGS_PATH="/etc/columnstore"
    DBRM_PATH="/var/lib/columnstore/data1/systemFiles/dbrm"
    STORAGEMANAGER_PATH="/var/lib/columnstore/storagemanager"
    STORAGEMANGER_CNF="$CS_CONFIGS_PATH/storagemanager.cnf"
    
    # can be overwrriten with parameter
    backup_location="/tmp/backups/"

**Pull request merged**